episodes/15-explanation.md

```diff
@@ -105,34 +124,67 @@ How might you prioritise what to keep if you have to cut it down?
-  - Header hierarchy (no h1 headers in the lesson body, no skipped levels e.g. h2 -> h4)
-  - Use descriptive link text - no "click here" or "this page" etc
-  - Image contrast \& tools to check this
+## Other Important Considerations for Lesson Text
+
+When writing your lesson text you also want to avoid unintentionally demotivating your learners.
+It is important to review your text for:
+
+- dismissive language - e.g. 'simply', 'just'
+- use of stereotypes (check learner profiles for stereotypes too)
+- expert awareness gaps, i.e. places where you may be assuming the learners know more than they actually do
+- fluid representations, i.e. using different terms with the same meaning interchangeably
+- unexplained or unnecessary jargon 
+- unexplained assumptions
+- sudden jumps in difficulty/complexity
+
+You should also review your text thinking about accessibility.
+This includes:
+
+  - Avoiding regional/cultural references and idioms that will not translate across borders/cultures
+  - Avoiding contractions
+  - Checking that all figures/images have well written alternative text, including [writing altnerative text for data visualizations](https://medium.com/nightingale/writing-alt-text-for-data-visualization-2a218ef43f81) - see [the Workbench documentation for directions on how to add alt-text and captions to images in a lesson](https://carpentries.github.io/sandpaper-docs/episodes.html#figures)
+  - Checking the header hierarchy - no h1 headers in the lesson body, no skipped levels
+  - [Using descriptive link text](https://www.imperial.ac.uk/staff/tools-and-reference/web-guide/training-and-events/materials/accessibility/links/) - no "click here" or "this page", etc
+  - [Checking the text and foreground contrast for images](https://contrastchecker.com/)
 
 ::::::::::::::::::::::::::::::::::::::  challenge
 
-## Exercise: Alternative text for images (5 minutes)
+## Optional Exercise: Alternative Text for Images (5 minutes)
+
+Which of the following is a good alt-text option for the image below?
 
-TODO/exclude to free up some time
+![Atmospheric Carbon Dioxide at Mauna Loa Observatory](https://gml.noaa.gov/webdata/ccgg/trends/co2_data_mlo.png){alt="Line graph of increasing carbon dioxide at the Mauna Loa Observatory from 1958 to present"}
+
+1. Graph of data
+2. Graph with increasing lines
+2. Line graph of increasing carbon dioxide in ppm at the Mauna Loa Observatory from 1958 to present
+3. Line graph of increasing carbon dioxide in ppm at the Mauna Loa Observatory, Hawaii, United States, from 1959 to present including values from each year. Red line shows variation in each year and black line is average for each year. 1959 = 315.90 ppm, 1960 = 316.91, 1961 = 317.64 ...
```

Comment:
Tried to include an overly descriptive one.  Not sure it is very realistic.  Would especially appreciate feedback on this point.

Reply:
> One other oddity, is that the data link right below this comment, isn't rendering in my local view. I tried removing the italics but it still wasn't rendering the link properly locally.

See https://github.com/carpentries/lesson-development-training/pull/42/files#r885675015